examples/gallery/plot/vectors.py

```diff
@@ -0,0 +1,59 @@
+"""
+Vectors
+------
+
+The :meth:`pygmt.Figure.plot` method can plot cartesian, circular, and geographic vectors.
+The `style` parameter controls vector attributes as in GMT6
+
+"""
+import numpy as np
+import pygmt
+
+# Create a plot with 15x15 cm basemap, Mercator projection (M) over the continental US
+region = [-127, -64, 24, 53]
+fig = pygmt.Figure()
+fig.coast(region=region, projection="M15c", B="10.0", N="1", A="4000", W="0.25p,black")
+
+
+# plot math angle arcs with different radii
+x = -110
+y = 37
+startdir = 90
+stopdir = 180
```

Comment:
```suggestion
startdir = 90 in degrees
stopdir = 180 in degrees
```

Reply:
Will also need a "#" here before "in degrees" to set comment